Winleafs.Wpf/Helpers/ToastHelper.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Windows;
+using ToastNotifications;
+using ToastNotifications.Lifetime;
+using ToastNotifications.Messages;
+using ToastNotifications.Position;
+using Winleafs.Wpf.Enums;
+
+namespace Winleafs.Wpf.Helpers
+{
+    public static class ToastHelper
+    {
+        public static void ShowNotification(string body, ToastLogLevel customLogLevel,
+            Corner corner = Corner.BottomRight, double secondsViewable = 3)
+        {
+            // App either hasn't started or doesn't have a main window
+            if (Application.Current?.MainWindow == null)
+            {
+                return;
+            }
+
+#pragma warning disable CA2000 // Dispose objects before losing scope
+            var notifier = new Notifier(configuration =>
```

Comment:
What happens if the main window is closed (but its instance still exists)?

Reply:
Reworked the logic to be more accurate and added Logging in case of failure